starfyre/__init__.py

```diff
@@ -8,15 +8,15 @@
 from .parser import RootParser
 
 
-def create_component(pyml="", css="", js="", client_side_python=""):
+def create_component(pyml="", css="", js="", client_side_python="", component_name=""):
     if client_side_python:
         new_js = transpile(client_side_python) + js
         js = new_js
 
     local_variables = inspect.currentframe().f_back.f_back.f_locals.copy()
     global_variables = inspect.currentframe().f_back.f_back.f_globals.copy()
 
-    parser = RootParser(local_variables, global_variables, css, js)
+    parser = RootParser(local_variables, global_variables, css, js, component_name)
```

Comment:
I have changed `class_name` to `component_name`. I believe this is better because it avoids potentially conflicting names and its name also perfectly describes what it does